src/main/markdown/doc/latest/tutorial/codeclient.md

```diff
@@ -26,26 +26,26 @@ You need a data structure to hold the list of stock symbols the user has entered
 
 1.  Create a data structure.
     *  In StockWatcher.java, in the StockWatcher class, create a new instance of a Java ArrayList.
-
-```
-public class StockWatcher implements EntryPoint {
-
-      private VerticalPanel mainPanel = new VerticalPanel();
-  private FlexTable stocksFlexTable = new FlexTable();
-  private HorizontalPanel addPanel = new HorizontalPanel();
-  private TextBox newSymbolTextBox = new TextBox();
-  private Button addStockButton = new Button("Add");
-  private Label lastUpdatedLabel = new Label();
-  private ArrayList<String> stocks = new ArrayList<String>();
     
-```
+    ```
```

Comment:
This doesn't fix the problem. The markdown engine has some difficulties to handle triple backticks inside lists. The best is to remove the triple backticks and just indent the block by 4*(n +1) (n being the depth of the list)